From patchwork Tue Jun  2 04:53:58 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 11583231
Return-Path: <SRS0=c+BJ=7P=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 9CFC1618
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue,  2 Jun 2020 04:54:06 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 8655E207D5
	for <patchwork-kvm@patchwork.kernel.org>;
 Tue,  2 Jun 2020 04:54:06 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (2048-bit key) header.d=canb.auug.org.au header.i=@canb.auug.org.au
 header.b="REQ1AbSt"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1726240AbgFBEyC (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Tue, 2 Jun 2020 00:54:02 -0400
Received: from lindbergh.monkeyblade.net ([23.128.96.19]:54796 "EHLO
        lindbergh.monkeyblade.net" rhost-flags-OK-OK-OK-OK) by vger.kernel.org
        with ESMTP id S1725890AbgFBEyC (ORCPT <rfc822;kvm@vger.kernel.org>);
        Tue, 2 Jun 2020 00:54:02 -0400
Received: from ozlabs.org (bilbo.ozlabs.org [IPv6:2401:3900:2:1::2])
        by lindbergh.monkeyblade.net (Postfix) with ESMTPS id F2E01C061A0E;
        Mon,  1 Jun 2020 21:54:01 -0700 (PDT)
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 49bfnz4bsxz9sSf;
        Tue,  2 Jun 2020 14:53:59 +1000 (AEST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1591073640;
        bh=rDyuOagNKw3935ar6u61XTbrq3VfmYxSPPFZvkTS/a4=;
        h=Date:From:To:Cc:Subject:From;
        b=REQ1AbSt8qb5RDz/3aD4wUoufWsmXCOg7kUvF6OzjOBK3U9XDYbI+aj1Vf+hduvZ7
         4VqC6R/nVcaoE32PxKpllqbrgVySqHjli1eA78utQN8r3/fptosamcymYGZfZC61N5
         /VrVWSxzArGm5fLr8CU21RiwwyYSBLf4EFrMsq7z+AvfV3j77oy88slU6OFDidD0EC
         r5aR82dVzi40drsLlzWt3hq2JESSFYzr7a2FA5qpdd4CvmosV7B8mrE9OhWQpnTgOL
         IAnFTQ0DeJD8S9if4nHL3gmoJlQwE7JRcPEbb6w0G9v3BAhbVt541MYKEom5oRdubF
         jR6Dy0cQZghdA==
Date: Tue, 2 Jun 2020 14:53:58 +1000
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paolo Bonzini <pbonzini@redhat.com>, KVM <kvm@vger.kernel.org>,
        Thomas Gleixner <tglx@linutronix.de>,
        Ingo Molnar <mingo@elte.hu>, "H. Peter Anvin" <hpa@zytor.com>,
        Peter Zijlstra <peterz@infradead.org>
Cc: Linux Next Mailing List <linux-next@vger.kernel.org>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        Vitaly Kuznetsov <vkuznets@redhat.com>
Subject: linux-next: manual merge of the kvm tree with the tip tree
Message-ID: <20200602145358.0a70f077@canb.auug.org.au>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm tree got a conflict in:

  arch/x86/kernel/kvm.c

between commit:

  a707ae1a9bbb ("x86/entry: Switch page fault exception to IDTENTRY_RAW")

from the tip tree and commit:

  68fd66f100d1 ("KVM: x86: extend struct kvm_vcpu_pv_apf_data with token info")

from the kvm tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc arch/x86/kernel/kvm.c
index 07dc47359c4f,d6f22a3a1f7d..000000000000
--- a/arch/x86/kernel/kvm.c
